src/Symfony/Bundle/FeatureToggleBundle/Resources/config/feature.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\FeatureToggle\FeatureChecker;
+use Symfony\Component\FeatureToggle\FeatureCheckerInterface;
+use Symfony\Component\FeatureToggle\FeatureCollection;
+
+return static function (ContainerConfigurator $container) {
+    $services = $container->services();
+
+    $services->set('toggle_feature.feature_collection', FeatureCollection::class)
```

Comment:
the service prefix should match the component name, i.e. `feature_toggle.feature_collection` same everywhere

Reply:
done